routes/api/budget.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+const express = require("express");
+const HttpStatus = require("http-status-codes");
+
+const interceptors = require("../interceptors");
+const models = require("../../models");
+
+const router = express.Router();
+
+router.get("/", async function (req, res) {
+  const row = await models.Budget.findAll();
+  res.json(row);
+});
+
+router.post("/", interceptors.requireLogin, async function (req, res) {
+  const user = req.user;
+  try {
+    let budget = await user.getBudget();
```

Comment:
Because the User.hasOne(models.Budget), we can now use a getBudget() method on the user object to get the Budget object (if it exists).